### ISSUE TITLE
installation problem

### ISSUE THREAD

**Issue by compumatter** [bejaneps/fb-google-events-sync#1]
Tried the following:
cd /usr/local/go/bin/fb-google-events-sync-master
sudo go build
main.go:9:2: cannot find package "github.com/huandu/facebook" in any of:
        /usr/lib/go-1.6/src/github.com/huandu/facebook (from $GOROOT)
        ($GOPATH not set)
jay@live:/usr/local/go/bin/fb-google-events-sync-master$

Seems to be referencing this guys package: https://github.com/huandu/facebook

**Comment by bejaneps** [comment]
Yes I'm sorry, as it is just demo version I didn't implement mod dependencies, so you have to download this package using go get tool:

go get github.com/huandu/facebook
